apps/ui/src/components/Ui/Button.vue

```diff
@@ -19,7 +19,7 @@ withDefaults(
   <button
     :type="type"
     :disabled="disabled || loading"
-    :class="primary && 'primary'"
+    :class="{ primary: primary, 'w-[46px] !px-0': loading }"
```

Comment:
Could we remove `!` so any component can pass overrides if required

Also we are passing `px-3.5`
<img width="866" alt="image" src="https://github.com/user-attachments/assets/f66be6ef-8374-48c9-a1d0-dc4f1e6ea810">

which is causing this jump
![Sep-17-2024 17-11-12](https://github.com/user-attachments/assets/b0ee83b7-a8d7-4236-a9cd-de6cbd61a0cb)

Reply:
```suggestion
    :class="{ primary: primary, 'w-[46px] px-0': loading }"
```
This works